example/generator/odp_generator.c

```diff
@@ -1333,6 +1376,13 @@ int main(int argc, char *argv[])
 
 	} else {
 		int cpu = odp_cpumask_first(&cpumask);
+		udp_args_t *udp_param = NULL;
+		uint16_t sport_range = args->appl.srcport_end -
+			args->appl.srcport + 1;
+		uint16_t dport_range = args->appl.dstport_end -
+			args->appl.dstport + 1;
+		float sport_step = (float)(sport_range) / num_workers;
```

Comment:
Is it possible to use integer math here?

Reply:
performance -  it is called only once before all the workers start, so is not a performance problem
functionality -  it is useful when you have less ports than workers (when you need  to split 3 ports to 7 workers)

Reply:
ack